block.c

```diff
@@ -1253,7 +1253,13 @@ static int mount_action(char *action, char *device, int type)
 	if (config_load(NULL))
 		return -1;
 
-	cache_load(1);
+	{
```

Comment:
Instead of using `{ ... }` please just declare additional variables on top of the function OR create a new
`static void cache_load_dev(const char *device)` function.

Reply:
Like that?  (This spin is neither compile nor run tested.)